### ISSUE TITLE
No I2C communication along with Connectivity Issues

### ISSUE THREAD

**Issue by Ian-cmd-ops** [runeharlyk/SpotMicroESP32-Leika#58]
```
--- Terminal on COM8 | 115200 8-N-1
--- Available filters and text transformations: colorize, debug, default, direct, esp32_exception_decoder, hexlify, log2file, nocontrol, printable, send_on_enter, time
--- More details at https://bit.ly/pio-monitor-filters
--- Quit: Ctrl+C | Menu: Ctrl+T | Help: Ctrl+T followed by Ctrl+H
�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx�x␀�x�x�x�����xx����␀��x␀���xx

rst:ets Jul 29 2019 12:21:46

rst:ets Jul 29 2019 12:21:46

rst:ets Jul 29 2019 12:21:46

rst:ets Jul 29 2019 12:21:46

rst:ets Jul 29 2019 12:21:46

rst:ets Jul 29 2019 12:21:46

rst:ets Jul 29 2019 12:21:46

rst:ets Jul 29 2019 12:21:46

rst:ets Jul 29 2019 12:21:46

rst:ets Jul 29 2019 12:21:46

rst:ets Jul 29 2019 12:21:46

rst:ets Jul 29 2019 12:21:46

rst:ets Jul 29 2019 12:21:46

rst:ets Jul 29 2019 12:21:46

rst:ets Jul 29 2019 12:21:46

rst:ets Jul 29 2019 12:21:46

rst:ets Jul 29 2019 12:21:46

rst:ets Jul 29 2019 12:21:46

rst:ets Jul 29 2019 12:21:46

rst:ets Jul 29 2019 12:21:46

rst:ets Jul 29 2019 12:21:46

rst:ets Jul 29 2019 12:21:46

rst:ets Jul 29 2019 12:21:46

rst:ets Jul 29 2019 12:21:46

rst:ets Jul 29 2019 12:21:46

rst:ets Jul 29 2019 12:21:46

rst:ets Jul 29 2019 12:21:46

rst:ets Jul 29 2019 12:21:46

rst:ets Jul 29 2019 12:21:46

rst:ets Jul 29 2019 12:21:46

rst:ets Jul 29 2019 12:21:46

rst:ets Jul 29 2019 12:21:46

rst:ets Jul 29 2019 12:21:46

rst:ets Jul 29 2019 12:21:46

rst:ets Jul 29 2019 12:21:46

rst:ets Jul 29 2019 12:21:46

rst:0x1 (POWERON_RESET),boot:0x13 (S0x1 (POWERON_RESET),boot:0x13 (S0x1 (POWERON_RESET),boot:0x13 (S0x1 (POWERON_RESET),boot:0x13 (S0x1 (POWERON_RESET),boot:0x13 (S0x1 (POWERON_RESET),boot:0x13 (S0x1 (POWERON_RESET),boot:0x13 (S0x1 (POWERON_RESET),boot:0x13 (S0x1 (POWERON_RESET),boot:0x13 (S0x1 (POWERON_RESET),boot:0x13 (S0x1 (POWERON_RESET),boot:0x13 (S0x1 (POWERON_RESET),boot:0x13 (S0x1 (POWERON_RESET),boot:0x13 (S0x1 (POWERON_RESET),boot:0x13 (S0x1 (POWERON_RESET),boot:0x13 (S0x1 (POWERON_RESET),boot:0x13 (S0x1 (POWERON_RESET),boot:0x13 (S0x1 (POWERON_RESET),boot:0x13 (S0x1 (POWERON_RESET),boot:0x13 (S0x1 (POWERON_RESET),boot:0x13 (S0x1 (POWERON_RESET),boot:0x13 (S0x1 (POWERON_RESET),boot:0x13 (S0x1 (POWERON_RESET),boot:0x13 (S0x1 (POWERON_RESET),boot:0x13 (S0x1 (POWERON_RESET),boot:0x13 (S0x1 (POWERON_RESET),boot:0x13 (S0x1 (POWERON_RESET),boot:0x13 (S0x1 (POWERON_RESET),boot:0x13 (S0x1 (POWERON_RESET),boot:0x13 (S0x1 (POWERON_RESET),boot:0x13 (S0x1 (POWERON_RESET),boot:0x13 (S0x1 (POWERON_RESET),boot:0x13 (S0x1 (POWERON_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:2
load:0x3fff0030,len:1184
load:0x40078000,len:13232
load:0x40080400,len:3028
entry 0x400805e4
[     9][D][esp32-hal-cpu.c:244] setCpuFrequencyMhz(): PLL: 480 / 2 = 240 Mhz, APB: 80000000 Hz
[    23][I][ESP32SvelteKit.cpp:81] begin(): [Running Firmware Version: %s] 0.0.1
[    38][D][WiFiGeneric.cpp:1039] _eventCallback(): Arduino Event: 0 - WIFI_READY
[   117][D][WiFiGeneric.cpp:1039] _eventCallback(): Arduino Event: 2 - STA_START
[   153][D][WiFiGeneric.cpp:1039] _eventCallback(): Arduino Event: 3 - STA_STOP
[   245][D][EventSocket.cpp:203] onSubscribe(): [EventSocket] onSubscribe for event: WiFiSettings
[   275][D][EventSocket.cpp:203] onSubscribe(): [EventSocket] onSubscribe for event: angles
[   276][D][EventSocket.cpp:203] onSubscribe(): [EventSocket] onSubscribe for event: ConfigurationSettings
[   300][I][esp32-hal-i2c.c:75] i2cInit(): Initialising I2C Master: sda=14 scl=15 freq=100000
[  1303][E][Wire.cpp:513] requestFrom(): i2cRead returned Error 263
[  2304][E][Wire.cpp:513] requestFrom(): i2cRead returned Error 263
[  3305][E][Wire.cpp:513] requestFrom(): i2cRead returned Error 263
[  4306][E][Wire.cpp:513] requestFrom(): i2cRead returned Error 263
[  5307][E][Wire.cpp:513] requestFrom(): i2cRead returned Error 263
[  6308][E][Wire.cpp:513] requestFrom(): i2cRead returned Error 263
[  7410][E][Wire.cpp:513] requestFrom(): i2cRead returned Error 263
[  8514][E][Wire.cpp:513] requestFrom(): i2cRead returned Error 263
[  8514][E][IMUService.h:35] begin(): [IMUService] MPU initialize failed
[  9516][E][Wire.cpp:513] requestFrom(): i2cRead returned Error 263
[ 10517][E][Wire.cpp:513] requestFrom(): i2cRead returned Error 263
[ 11518][E][Wire.cpp:513] requestFrom(): i2cRead returned Error 263
[ 12519][E][Wire.cpp:513] requestFrom(): i2cRead returned Error 263
[ 12520][W][Wire.cpp:301] begin(): Bus already started in Master Mode.
[ 13532][E][Wire.cpp:513] requestFrom(): i2cRead returned Error 263
[ 13539][D][EventSocket.cpp:203] onSubscribe(): [EventSocket] onSubscribe for event: servoConfiguration
[ 14569][E][Wire.cpp:513] requestFrom(): i2cRead returned Error 263
[ 15577][E][Wire.cpp:513] requestFrom(): i2cRead returned Error 263
[ 15583][I][ServoController.h:169] configure(): [ServoController] Configured with oscillator frequency 27000000 and PWM frequency 50
[ 15585][I][APSettingsService.cpp:89] startAP(): [APSettingsService] Starting software access point
[ 15600][D][WiFiGeneric.cpp:1039] _eventCallback(): Arduino Event: 0 - WIFI_READY
[ 15605][D][WiFiGeneric.cpp:1039] _eventCallback(): Arduino Event: 10 - AP_START
[ 16178][D][WiFiGeneric.cpp:1039] _eventCallback(): Arduino Event: 11 - AP_STOP
[ 16178][D][WiFiGeneric.cpp:1039] _eventCallback(): Arduino Event: 10 - AP_START
[ 16181][I][APSettingsService.cpp:98] startAP(): [APSettingsService] Starting captive portal on 192.168.4.1
[ 29979][D][WiFiGeneric.cpp:1039] _eventCallback(): Arduino Event: 12 - AP_STACONNECTED
[ 30085][D][WiFiGeneric.cpp:1039] _eventCallback(): Arduino Event: 14 - AP_STAIPASSIGNED
[ 52308][I][PsychicHttpServer.cpp:235] openCallback(): [psychic] New client connected 52
[ 52376][I][PsychicHttpServer.cpp:261] closeCallback(): [psychic] Client disconnected 52
[ 52408][I][PsychicHttpServer.cpp:235] openCallback(): [psychic] New client connected 52
[ 52478][I][PsychicHttpServer.cpp:261] closeCallback(): [psychic] Client disconnected 52
[ 66240][I][PsychicHttpServer.cpp:235] openCallback(): [psychic] New client connected 52
[ 66298][I][PsychicHttpServer.cpp:261] closeCallback(): [psychic] Client disconnected 52
[ 67796][I][PsychicHttpServer.cpp:235] openCallback(): [psychic] New client connected 52
[ 67858][I][PsychicHttpServer.cpp:261] closeCallback(): [psychic] Client disconnected 52
[ 68103][I][PsychicHttpServer.cpp:235] openCallback(): [psychic] New client connected 52
[ 68163][I][PsychicHttpServer.cpp:261] closeCallback(): [psychic] Client disconnected 52
[ 73209][I][PsychicHttpServer.cpp:235] openCallback(): [psychic] New client connected 52
[ 73295][I][PsychicHttpServer.cpp:261] closeCallback(): [psychic] Client disconnected 52
[ 73974][I][PsychicHttpServer.cpp:235] openCallback(): [psychic] New client connected 52
[ 74120][I][PsychicHttpServer.cpp:235] openCallback(): [psychic] New client connected 53
[ 75150][I][PsychicHttpServer.cpp:235] openCallback(): [psychic] New client connected 54
[ 75152][I][PsychicHttpServer.cpp:235] openCallback(): [psychic] New client connected 55
[ 75496][I][PsychicHttpServer.cpp:235] openCallback(): [psychic] New client connected 56
[ 75504][I][EventSocket.cpp:69] onWSOpen(): [EventSocket] ws[192.168.4.2][56] connect

abort() was called at PC 0x4015a737 on core 1


Backtrace: 0x4008390d:0x3ffbc160 0x4008ee35:0x3ffbc180 0x40094401:0x3ffbc1a0 0x4015a737:0x3ffbc220 0x4015a77e:0x3ffbc240 0x4015a6df:0x3ffbc260 0x400e2fe3:0x3ffbc280 0x400e4067:0x3ffbc2a0 0x40189575:0x3ffbc390 0x400d9726:0x3ffbc3b0 0x400d9f39:0x3ffbc470 0x401175c1:0x3ffbc4f0 0x40117e56:0x3ffbc590 0x40116b24:0x3ffbc5b0 0x4018a782:0x3ffbc5d0 0x40116c49:0x3ffbc5f0

  #0  0x4008390d:0x3ffbc160 in panic_abort at /Users/ficeto/Desktop/ESP32/ESP32S2/esp-idf-public/components/esp_system/panic.c:408
  #1  0x4008ee35:0x3ffbc180 in esp_system_abort at /Users/ficeto/Desktop/ESP32/ESP32S2/esp-idf-public/components/esp_system/esp_system.c:137
  #2  0x40094401:0x3ffbc1a0 in abort at /Users/ficeto/Desktop/ESP32/ESP32S2/esp-idf-public/components/newlib/abort.c:46
  #3  0x4015a737:0x3ffbc220 in __cxxabiv1::__terminate(void (*)()) at /builds/idf/crosstool-NG/.build/HOST-x86_64-w64-mingw32/xtensa-esp32-elf/src/gcc/libstdc++-v3/libsupc++/eh_terminate.cc:47
  #4  0x4015a77e:0x3ffbc240 in std::terminate() at /builds/idf/crosstool-NG/.build/HOST-x86_64-w64-mingw32/xtensa-esp32-elf/src/gcc/libstdc++-v3/libsupc++/eh_terminate.cc:57
  #5  0x4015a6df:0x3ffbc260 in __cxa_throw at /builds/idf/crosstool-NG/.build/HOST-x86_64-w64-mingw32/xtensa-esp32-elf/src/gcc/libstdc++-v3/libsupc++/eh_throw.cc:95
  #6  0x400e2fe3:0x3ffbc280 in char_to_message_type(char) at lib/ESP32-sveltekit/EventSocket.cpp:13 (discriminator 2)
  #7  0x400e4067:0x3ffbc2a0 in EventSocket::onFrame(PsychicWebSocketRequest*, httpd_ws_frame*) at lib/ESP32-sveltekit/EventSocket.cpp:91
  #8  0x40189575:0x3ffbc390 in int std::__invoke_impl<int, int (EventSocket::*&)(PsychicWebSocketRequest*, httpd_ws_frame*), EventSocket*&, PsychicWebSocketRequest*, httpd_ws_frame*>(std::__invoke_memfun_deref, int (EventSocket::*&)(PsychicWebSocketRequest*, httpd_ws_frame*), EventSocket*&, PsychicWebSocketRequest*&&, httpd_ws_frame*&&) at c:\users\rip_h\.platformio\packages\toolchain-xtensa-esp32\xtensa-esp32-elf\include\c++\8.4.0\bits/invoke.h:73
      (inlined by) std::__invoke_result<int (EventSocket::*&)(PsychicWebSocketRequest*, httpd_ws_frame*), EventSocket*&, PsychicWebSocketRequest*, httpd_ws_frame*>::type std::__invoke<int (EventSocket::*&)(PsychicWebSocketRequest*, httpd_ws_frame*), EventSocket*&, PsychicWebSocketRequest*, httpd_ws_frame*>(int (EventSocket::*&)(PsychicWebSocketRequest*, httpd_ws_frame*), EventSocket*&, PsychicWebSocketRequest*&&, httpd_ws_frame*&&) at c:\users\rip_h\.platformio\packages\toolchain-xtensa-esp32\xtensa-esp32-elf\include\c++\8.4.0\bits/invoke.h:95
      (inlined by) int std::_Bind<int (EventSocket::*(EventSocket*, std::_Placeholder<1>, std::_Placeholder<2>))(PsychicWebSocketRequest*, httpd_ws_frame*)>::__call<int, PsychicWebSocketRequest*&&, httpd_ws_frame*&&, 0u, 1u, 2u>(std::tuple<PsychicWebSocketRequest*&&, httpd_ws_frame*&&>&&, std::_Index_tuple<0u, 1u, 2u>) at c:\users\rip_h\.platformio\packages\toolchain-xtensa-esp32\xtensa-esp32-elf\include\c++\8.4.0/functional:400
      (inlined by) int std::_Bind<int (EventSocket::*(EventSocket*, std::_Placeholder<1>, std::_Placeholder<2>))(PsychicWebSocketRequest*, httpd_ws_frame*)>::operator()<PsychicWebSocketRequest*, httpd_ws_frame*, int>(PsychicWebSocketRequest*&&, httpd_ws_frame*&&) at c:\users\rip_h\.platformio\packages\toolchain-xtensa-esp32\xtensa-esp32-elf\include\c++\8.4.0/functional:484
      (inlined by) std::_Function_handler<int (PsychicWebSocketRequest*, httpd_ws_frame*), std::_Bind<int (EventSocket::*(EventSocket*, std::_Placeholder<1>, std::_Placeholder<2>))(PsychicWebSocketRequest*, httpd_ws_frame*)> >::_M_invoke(std::_Any_data const&, PsychicWebSocketRequest*&&, httpd_ws_frame*&&) at c:\users\rip_h\.platformio\packages\toolchain-xtensa-esp32\xtensa-esp32-elf\include\c++\8.4.0\bits/std_function.h:282
  #9  0x400d9726:0x3ffbc3b0 in std::function<int (PsychicWebSocketRequest*, httpd_ws_frame*)>::operator()(PsychicWebSocketRequest*, httpd_ws_frame*) const at c:\users\rip_h\.platformio\packages\toolchain-xtensa-esp32\xtensa-esp32-elf\include\c++\8.4.0\bits/std_function.h:687
      (inlined by) PsychicWebSocketHandler::handleRequest(PsychicRequest*) at lib/PsychicHttp/src/PsychicWebSocket.cpp:208
  #10 0x400d9f39:0x3ffbc470 in PsychicEndpoint::requestCallback(httpd_req*) at lib/PsychicHttp/src/PsychicEndpoint.cpp:72
  #11 0x401175c1:0x3ffbc4f0 in httpd_req_new at /Users/ficeto/Desktop/ESP32/ESP32S2/esp-idf-public/components/esp_http_server/src/httpd_parse.c:776
  #12 0x40117e56:0x3ffbc590 in httpd_sess_process at /Users/ficeto/Desktop/ESP32/ESP32S2/esp-idf-public/components/esp_http_server/src/httpd_sess.c:419 (discriminator 15)
  #13 0x40116b24:0x3ffbc5b0 in httpd_process_session at /Users/ficeto/Desktop/ESP32/ESP32S2/esp-idf-public/components/esp_http_server/src/httpd_main.c:178   
      (inlined by) httpd_process_session at /Users/ficeto/Desktop/ESP32/ESP32S2/esp-idf-public/components/esp_http_server/src/httpd_main.c:163
  #14 0x4018a782:0x3ffbc5d0 in httpd_sess_enum at /Users/ficeto/Desktop/ESP32/ESP32S2/esp-idf-public/components/esp_http_server/src/httpd_sess.c:50 (discriminator 1)
  #15 0x40116c49:0x3ffbc5f0 in httpd_server at /Users/ficeto/Desktop/ESP32/ESP32S2/esp-idf-public/components/esp_http_server/src/httpd_main.c:228
      (inlined by) httpd_thread at /Users/ficeto/Desktop/ESP32/ESP32S2/esp-idf-public/components/esp_http_server/src/httpd_main.c:250




ELF file SHA256: a047969d52ee0992

Rebooting...
ets Jul 29 2019 12:21:46

rst:0xc (SW_CPU_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:2
load:0x3fff0030,len:1184
load:0x40078000,len:13232
load:0x40080400,len:3028
entry 0x400805e4
[    36][D][esp32-hal-cpu.c:244] setCpuFrequencyMhz(): PLL: 480 / 2 = 240 Mhz, APB: 80000000 Hz
[    50][I][ESP32SvelteKit.cpp:81] begin(): [Running Firmware Version: %s] 0.0.1
[    65][D][WiFiGeneric.cpp:1039] _eventCallback(): Arduino Event: 0 - WIFI_READY
[   144][D][WiFiGeneric.cpp:1039] _eventCallback(): Arduino Event: 2 - STA_START
[   180][D][WiFiGeneric.cpp:1039] _eventCallback(): Arduino Event: 3 - STA_STOP
[   271][D][EventSocket.cpp:203] onSubscribe(): [EventSocket] onSubscribe for event: WiFiSettings
[   301][D][EventSocket.cpp:203] onSubscribe(): [EventSocket] onSubscribe for event: angles
[   303][D][EventSocket.cpp:203] onSubscribe(): [EventSocket] onSubscribe for event: ConfigurationSettings
[   326][I][esp32-hal-i2c.c:75] i2cInit(): Initialising I2C Master: sda=14 scl=15 freq=100000
[  1329][E][Wire.cpp:513] requestFrom(): i2cRead returned Error 263
[  2330][E][Wire.cpp:513] requestFrom(): i2cRead returned Error 263
[  3331][E][Wire.cpp:513] requestFrom(): i2cRead returned Error 263
[  4332][E][Wire.cpp:513] requestFrom(): i2cRead returned Error 263
[  5333][E][Wire.cpp:513] requestFrom(): i2cRead returned Error 263
[  6334][E][Wire.cpp:513] requestFrom(): i2cRead returned Error 263
[  7436][E][Wire.cpp:513] requestFrom(): i2cRead returned Error 263
[  8540][E][Wire.cpp:513] requestFrom(): i2cRead returned Error 263
[  8541][E][IMUService.h:35] begin(): [IMUService] MPU initialize failed
[  9542][E][Wire.cpp:513] requestFrom(): i2cRead returned Error 263
[ 10543][E][Wire.cpp:513] requestFrom(): i2cRead returned Error 263
[ 11544][E][Wire.cpp:513] requestFrom(): i2cRead returned Error 263
[ 12545][E][Wire.cpp:513] requestFrom(): i2cRead returned Error 263
[ 12546][W][Wire.cpp:301] begin(): Bus already started in Master Mode.
[ 13558][E][Wire.cpp:513] requestFrom(): i2cRead returned Error 263
[ 13565][D][EventSocket.cpp:203] onSubscribe(): [EventSocket] onSubscribe for event: servoConfiguration
[ 14596][E][Wire.cpp:513] requestFrom(): i2cRead returned Error 263
[ 15603][E][Wire.cpp:513] requestFrom(): i2cRead returned Error 263
[ 15609][I][ServoController.h:169] configure(): [ServoController] Configured with oscillator frequency 27000000 and PWM frequency 50
[ 15611][I][APSettingsService.cpp:89] startAP(): [APSettingsService] Starting software access point
[ 15626][D][WiFiGeneric.cpp:1039] _eventCallback(): Arduino Event: 0 - WIFI_READY
[ 15631][D][WiFiGeneric.cpp:1039] _eventCallback(): Arduino Event: 10 - AP_START
[ 16204][D][WiFiGeneric.cpp:1039] _eventCallback(): Arduino Event: 11 - AP_STOP
[ 16205][D][WiFiGeneric.cpp:1039] _eventCallback(): Arduino Event: 10 - AP_START
[ 16208][I][APSettingsService.cpp:98] startAP(): [APSettingsService] Starting captive portal on 192.168.4.1
```
I wanted to let know that the AP connectivity is a bit dodgy. I can't get the I2C communication for any number of reasons but I think I was suppose to put the 4.7k resistor inline for the SDA and SCL connections.

**Comment by runeharlyk** [comment]
I will look into the connectivity. For the I2C, there should not be anything between SCL and SDA connections.

**Comment by Ian-cmd-ops** [comment]
![image](https://github.com/runeharlyk/SpotMicroESP32-Leika/assets/78785932/100a4ebb-261a-4467-98dc-e2a24c6076d0)


> I will look into the connectivity. For the I2C, there should not be anything between SCL and SDA connections.

https://a.co/d/037ObbSe
I'm using this serial wombat as an I2C bus. I can't get read any serial data off the board when it's connected. do you have diagram to read the info from the board if I have a USB to TTL adapter.

**Comment by runeharlyk** [comment]
The Serial Wombat you linked to is not a device in of it self, the description mentions *Board contains no electronics*. Its only made to make it simpler to hook up other devices - like a breadboard. You have to connect the board to the esp32, and then the I2C devices (MPU6050, PCA9685 etc) to the "Serial Wombat".

**Comment by Ian-cmd-ops** [comment]
For clarification, I have connected that i2c and connected the pins in the same order as the silk screen. I'm unable to verify i2c data when it was connected. I had no comms with IMU when connected to the AP with the IMU connected to the Serial Wombat. Additional I couldn't verify comms with PCA9685. I thought it was my wiring was the issue but I triple checked it. I've exhausted most possible troubleshooting issues from my end. I'll attempt to see it the issue continues on my breadboard. Cheers mate.
